import.ftd

```diff
@@ -7,7 +7,7 @@ id: outer
 
 -- ft.markdown:
 
-`ftd` code in one page/component gains access to the code in another page by the process of importing it. In `ftd` language the uses of `import` are two typed :
+`ftd` code in one page/component gains access to the code in another page by the process of importing it. It takes input like this :
```

Comment:
@arcoprova the lines are too long. You have to manually put line breaks after 80 characters or so per line.

Reply:
@arcoprova as advised use 80 chars limitations for all lines

Reply:
ok sir